docker/Dockerfile

```diff
@@ -1,13 +1,20 @@
-FROM node:8
+FROM node:8.14.0-alpine as basis
 
-WORKDIR /opt/subscription-manager
-EXPOSE 80
-CMD ["npm", "run", "subscription"]
+WORKDIR /opt/data-broker
+
+RUN apk add git python make bash gcc g++ --no-cache
 
 COPY package.json .
 COPY package-lock.json .
-RUN npm install
 
+RUN npm install
 COPY . .
```

Comment:
just a suggestion, instead copy everything, just copy the code

Reply:
I think that this way is easier and less prone to errors. Copying just the code is not enough - we might need to copy also tsconfig, tslint and other things (I think that only the documentation and dockerfile would be left out). I agree that it would be better not to copy the .git (and all git-related stuff for that matter) to the image, as this might increase the image size significantly, but these we can add to .dockerignore file later.